website/content/v1.6/advanced/building-images.md

```diff
@@ -33,6 +33,14 @@ make <target> PLATFORM=linux/arm64 # build for arm64 only
 make <target> PLATFORM=linux/arm64,linux/amd64 # build for arm64 and amd64, container images will be multi-arch
 ```
 
+## Custom `PKGS`
+
+When [customizing Linux kernel]({{< relref "customizing-the-kernel" >}}), the source for the [`siderolabs/pkgs`](https://github.com/siderolabs/pkgs) repository can
+be overridden with:
+
+* if you built and pushed only a custom `kernel` package, the reference can be overridden with `PKG_KERNEL` variable: `make <target> PKG_KERNEL=<registry>/<username>/kernel:<tag>`
```

Comment:
i think this has the downside that the user also needs to build a arm64 image if building imager?

Reply:
okay, saw the installer `INSTALER_ARCH` comment below, i guess this can be ignored

Reply:
no, because `INSTALLER_ARCH=targetarch`